data/header.json

```diff
@@ -13,5 +13,6 @@
     },
     "block_merkle_tree_root":"MERKLE_COMM~yB4_Aqa35_PoskgTpcCR1oVLh6BUdLHIs7erHKWi-usUAAAAAAAAAAEAAAAAAAAAJg",
     "fee_merkle_tree_root":"MERKLE_COMM~VJ9z239aP9GZDrHp3VxwPd_0l28Hc5KEAB1pFeCIxhYgAAAAAAAAAAIAAAAAAAAAdA",
-    "fee_info":{"account":"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266","amount":"0x0"}
+    "fee_info":{"account":"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266","amount":"0x0"},
+    "chain_config":{"chain_config":{"Left":{"chain_id":"0x8a19","max_block_size":10240,"base_fee":"0x0"}}}
```

Comment:
redudant "chain_config" here

Reply:
Sorry @ImJeremyHe what is `untag`?

Reply:
Sorry, I mean we could customize the `serde` using this https://serde.rs/enum-representations.html#untagged